app/(header-default)/dirigeants/[slug]/_component/sections/entreprise/dirigeants-content.tsx

```diff
@@ -1,23 +1,71 @@
+import FAQLink from '#components-ui/faq-link';
 import { SeePersonPageLink } from '#components-ui/see-personn-page-link';
 import { FullTable } from '#components/table/full';
 import { IUniteLegale } from '#models/core/types';
-import { IDirigeants, IEtatCivil, IPersonneMorale } from '#models/rne/types';
+import {
+  IDirigeantsWithMetadata,
+  IEtatCivil,
+  IPersonneMorale,
+} from '#models/rne/types';
 import { formatDateLong, formatDatePartial, formatIntFr } from '#utils/helpers';
 import { isPersonneMorale } from '../is-personne-morale';
 
 type IDirigeantContentProps = {
-  dirigeants: IDirigeants;
+  dirigeants: IDirigeantsWithMetadata;
   uniteLegale: IUniteLegale;
 };
 
-export function DirigeantContent({
+const dataSourceTooltip = ({
+  dataType,
+  isInIg,
+  isInInpi,
+}: {
+  dataType: string;
+  isInIg?: boolean;
+  isInInpi?: boolean;
+}) => {
+  if (!isInIg && !isInInpi) {
+    return <></>;
+  }
+
+  return (
+    <>
+      {!isInIg && (
+        <>
+          {' '}
+          <FAQLink tooltipLabel={<></>}>
+            Ce {dataType} n‘apparait pas dans les données d‘Infogreffe.
+          </FAQLink>
+        </>
+      )}
+      {!isInInpi && (
+        <>
+          {' '}
+          <FAQLink tooltipLabel={<></>}>
```

Comment:
<img width="1183" alt="image" src="https://github.com/user-attachments/assets/9bb71775-d401-4b65-9e0f-08655d483f08">

New version

Reply:
<img width="341" alt="image" src="https://github.com/user-attachments/assets/bcd86348-3a5c-4c68-9b98-e1a87ce7f537">

Maybe use a sentence that explain the issue rather than underline the whole line